cmd/main.go

```diff
@@ -1,45 +1,47 @@
 package main
 
 import (
-	"log"
 	"os"
+	"time"
+
+	"github.com/rs/zerolog"
 
 	"github.com/unagex/kondense/pkg/controller"
 	"github.com/unagex/kondense/pkg/utils"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
 func main() {
-	// create logger.
-	l := log.Default()
-	l.SetFlags(log.Lshortfile | log.LstdFlags)
+	l := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr, TimeFormat: time.RFC3339}).Level(zerolog.TraceLevel).With().Timestamp().Logger()
+
+	//l.SetFlags(log.Lshortfile | log.LstdFlags)
 
 	// get pod name and namespace.
 	name := os.Getenv("HOSTNAME")
 	namespaceByte, err := os.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/namespace")
 	if err != nil {
-		l.Fatal(err)
+		l.Err(err)
```

Comment:
`l.Fatal` log the error then exit. 
`l.Err()` do nothing.

All `l.Fatal` should be replaced by the same thing and it's not the case in this file.